source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,520 @@
+
+For older versions, see the `releases`_ page.
+
+Although not required, it is recommended to download the corresponding 
+checksum file (designated with `-SHA512SUMS` at the end of the filename) 
+for the image in order to verify its integrity.
+
+.. _image: https://download.clearlinux.org/image
+.. _releases: https://download.clearlinux.org/releases
+
+.. _verify-clr-image-for-vmware:
+
+Verify the integrity of the downloaded image (recommended)
+==========================================================
+
+.. note::
+
+  The command-line examples below reference an `installer ISO` image file.  
+  Substitute it for the desired type of image file to verify.  
+
+* For Linux distros and macOS:
+
+  #.  Start a terminal emulator.
+  #.  Go to the directory with the downloaded file(s).
+  #.  To verify the integrity of the image and compare it to its original checksum, 
+      enter:
 
       .. code-block:: console
 
-         $ xz -d clear-{release-number}-live.img.xz
+        $ sha512sum ./clear-<version>-installer.iso.xz | diff ./clear-<version>-installer.iso.xz-SHA512SUMS -
+
+      If the checksum of the downloaded image is different than the original
+      checksum, the differences will displayed. An empty output indicates a match.
+
+* For Windows:
 
-   * On Windows, use `7zip`_.
+  #.  Start Command-Prompt.
+  #.  Go to the directory with the downloaded files.
+  #.  To verify the integrity of the image, enter:
 
-     - Right-click the file and select
-       :menuselection:`Extract in the same directory`.
+      .. code-block:: console
+
+        C:\> CertUtil -hashfile ./clear-<version>-installer.iso.xz sha512
 
-       .. image:: ./figures/7zipwin.png
-          :alt: 7zip extract here command
+      Manually compare the output with the original checksum value shown in 
+      the downloaded checksum file (i.e. `clear-<version>-installer.iso.xz-SHA512SUMS`) 
+      to make sure they match.
 
-#. Create a virtual machine with the following configuration:
+.. _uncompress-image-for-vmware:
 
-   * :guilabel:`Guest OS`: Linux, Distribution: Other 3.x Linux (64-bits)
+Uncompress the image
+====================
 
-   * :guilabel:`UEFI support`: |CL| uses `systemd-boot` as the UEFI
-     boot manager for EFI images. Add UEFI support. Find this option's
-     settings using the vSphere GUI. Go to the
-     :menuselection:`Configuration settings of the virtual machine-->General`
-     Tab and select :guilabel:`EFI boot firmware`.
+.. note::
 
-   * :guilabel:`SCSI Para-virtualized disk`: Convert to
-      :abbr:`VMDK (VMware DisK)` and attach the |CL| image downloaded above.
+  The command-line examples below reference an `installer ISO` image file.  
+  Substitute it for the desired type of image file to uncompress.  
 
-      + On Linux, To convert Clear Linux image to VMDK\* you can use the
-        :command:`qemu-img` command:
+* For Linux distros (an installer ISO image is used as an example):
```

Comment:
Do not use "For" the correct preposition here would be "On". My previous comment holds true. All the steps for downloading, verifying, and extracting the image should be in a single section per OS. Ideally, outside of the tutorial and the tutorial can then reuse the content. This mixture is confusing and forces Linux users to read Mac and Windows instructions, for example.